assets/js/util/cache-data.js

```diff
@@ -107,7 +107,7 @@ rl.question( 'Username: ', ( username ) => {
 				];
 				for ( const scriptName of scriptsToMatch ) {
 					// A regex that looks for the localized variable.
-					const matchStringRegex = '[window.|var ]' + scriptName + ' = (.*)';
+					const matchStringRegex = '[global.|var ]' + scriptName + ' = (.*)';
```

Comment:
This looks like one exception we should keep since it's a pattern matching in html. It probably doesn't even need the lint rule ignored here since it's not declaring/accessing a variable. Also, I think this pattern is incorrect as it's using a character class `[]` rather than a grouping `()`.
 ```suggestion
					const matchStringRegex = '(?:window\.|var )' + scriptName + ' = (.*)';
```

Reply:
Oops, you're right! Shape this didn't fail a test, but yes: that was overzealous find+replace.